README.md

```diff
@@ -215,6 +215,9 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools which support continuously building, testing and releasing applications.*
 
 * [Bamboo ![c]](https://www.atlassian.com/software/bamboo) - Atlassian's solution with good integration of their other products. You can either apply for an open-source license or buy it.
+* [Buddy ![c]](https://buddy.works/) - Hosted CI/CD platform with on-premise option.
+* [CircleCI ![c]](https://circleci.com) - Hosted CI/CD platform with docker and OSX support. Concurrent builds rather than project based billing model.
```

Comment:
@Moshe-Immerman By "OSX", do you mean "Mac OS X"? Where does it say on the website that CircleCI supports "OSX"? How does it do so? Why is this useful for Java developers? And if you did mean "Mac OS X", how about saying "macOS" (the new official name) instead?